### DISCUSSION TITLE
【委托】求好看的黑塔头像つ♥゛

### DISCUSSION THREAD

**Discussion by wenkui921** [share121/inter-knot#3442] [其他]
本人素黑塔厨，想要用好看的黑塔图当头像ヾ(°∇°)ﾉﾞ
![1724265670570cfbe2eaad4b9caca19bc182a0e3e7e09c4a37d1bddec8b4788334b3c413646d1 0_edit_504843370765674](https://github.com/user-attachments/assets/cd73c511-ce0e-4dca-9d5c-9e5a99e2a920)
拜托了( ⁼̴ω⁼̴)人゛

**Comment by shi423** [comment]
![98650216-0](https://github.com/user-attachments/assets/5826cc88-2797-4a0f-9b6d-e799b28eba09)

**Reply by wenkui921** [reply]
好看
![1724265670570cfbe2eaad4b9caca19bc182a0e3e7e09c4a37d1bddec8b4788334b3c413646d1 0_edit_504843370765674](https://github.com/user-attachments/assets/9a16ca69-60c5-47eb-9eb2-b8d9d9d8e71d)
